pkg/command/root.go

```diff
@@ -5,11 +5,11 @@
 package command
 
 import (
-	"errors"
 	"fmt"
 	"os"
 	"strings"
 
+	"github.com/pkg/errors"
```

Comment:
Actually, is this change ok?

Reply:
That should be fine. But let me revert it to the actual package(as there is no change that requires change in package)